src/devices/SenseHat/SenseHatLedMatrixSysFs.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Device.Gpio;
+using System.Diagnostics;
+using System.Drawing;
+using System.Runtime.CompilerServices; 
+
+namespace Iot.Device.SenseHat
+{
+    public class SenseHatLedMatrixSysFs : SenseHatLedMatrix
```

Comment:
Is this specific to SysFs driver only or generic UnixDriver

Reply:
not sure I understand - SenseHat driver exposes its kernel objects using sysfs. This implementation requires the SenseHat driver - I2C does not. This implementation will produce better colors than the I2C

Reply:
Ok, seems sysfs specific